tasks/build-options/validateConfigs.js

```diff
@@ -47,13 +53,15 @@ async function main () {
     validateFile(filePath)
   }
   if (invalidJsonFiles.length) {
+    if (argv.mode === 'verbose') console.warn(`${prog}: Invalid JSON files: ${invalidJsonFiles}`)
```

Comment:
I don't think this log is required since at this point we are throwing an error already.

Reply:
I want verbose mode to output the bad files themselves, not just the number of bad files. I guess i could throw the error in an else statement but it's verbose mode so I figured there was no harm in outputting both.